Webmin/input/auxiliary/DISCLAIMER

```diff
@@ -0,0 +1,16 @@
+The upstream STIG for RHEL 6 Server profile "stig-rhel6-server-upstream" is
+developed under the DoD consensus model and DISA FSO Vendor STIG process,
+serving as the upstream development environment for the Red Hat Enterprise
+Linux 6 Server STIG.
+
+As a result of the upstream/downstream relationship between the SCAP Security
+Guide project and the official DISA FSO STIG baseline, users should expect
+variance between SSG and DISA FSO content. For official DISA FSO STIG content,
+refer to http://iase.disa.mil/stigs/os/unix/red_hat.html.
+
+While this profile is packaged by Red Hat as part of the SCAP Security Guide
+package, please note that commercial support of this SCAP content is NOT
+available. This profile is provided as example SCAP content with no endorsement
+for suitability or production readiness. Support for this profile is provided
+by the upstream SCAP Security Guide community on a best-effort basis. The
+upstream project homepage is https://fedorahosted.org/scap-security-guide/.
```

Comment:
This file can be dropped for Webmin, at least until a formal FSO relationship forms to create a formal Webmin STIG.